src/envoy/auth/jwt.cc

```diff
@@ -277,10 +277,24 @@ Jwt::Jwt(const std::string &jwt) {
   }
 
   iss_ = payload_->getString("iss", "");
-  aud_ = payload_->getString("aud", "");
   sub_ = payload_->getString("sub", "");
   exp_ = payload_->getInteger("exp", 0);
 
+  // "aud" can be either string array or string.
+  // Try as string array, read it as empty array if doesn't exist.
+  try {
+    aud_ = payload_->getStringArray("aud", true);
+  } catch (...) {
```

Comment:
Do not catch all exceptions but `Json::Exception` only.

Reply:
Done